packages/manager/src/features/CloudPulse/Dashboard/CloudPulseDashboard.tsx

```diff
@@ -105,12 +107,16 @@ export const CloudPulseDashboard = (props: DashboardProperties) => {
     Boolean(resourceList)
   );
 
+  if (isResourcesApiError) {
+    return renderErrorState('Failed to fetch Resources');
+  }
+
+  if (isDashboardApiError) {
+    return renderErrorState('Failed to fetch the dashboard details');
+  }
+
```

Comment:
```suggestion

  if (isDashboardApiError) {
    return renderErrorState('Failed to fetch the dashboard details');
  }
  
  if (isResourcesApiError) {
    return renderErrorState('Failed to fetch Resources');
  }
```

lets keep the Dashboard error check first as resource is depended on service type.

Reply:
sure, i have done, by the order doesn't matter here, as useQuery hooks cannot be conditioned. we have reached here after all the useQuery executions